pages/content/amp-dev/documentation/components/reference/amp-ad-v0.1@tr.md

```diff
@@ -10,7 +11,7 @@ teaser:
 
 Bir reklamın görüntüleneceği kapsayıcı. `amp-embed`, `amp-ad` etiketinin bir diğer adıdır ve bu etiketin tüm işlevlerini farklı bir etiket adıyla alır. Anlam açısından daha doğru olduğunda `amp-embed` etiketini kullanın. AMP dokümanları yalnızca HTTPS aracılığıyla sunulan reklamları/yerleştirmeleri destekler.
 
-# `amp-ad` / `amp-embed`
+
```

Comment:
The title above and not this one should have been removed.

Reply:
done

Reply:
You got me wrong here.
Please have a look at another language translation of the same document.
The first h1 should have been removed. The second after the first paragraph should be kept.
And the title in the frontmatter should start with `$title: `

Reply:
done